### DISCUSSION TITLE
Multiple controller profiles?

### DISCUSSION THREAD

**Discussion by Vin-bin** [project64/project64#2151] [Ideas]
Hi, I was thinking it would be nice if there was the option to create multiple controller profiles that you could switch between for different games.

**Comment by IronYoshi64** [comment]
You already can do this, but it's not with the default Input plugin (the one with N64 controller picture), at least not right now. You need to use NRage Input plugin to do that. You can Save and Load Profiles with the buttons at the top of the UI.
